geoportailv3/static/js/maincontroller.js

```diff
@@ -211,6 +219,30 @@ app.MainController = function(
       $('app-themeswitcher #themes-content').collapse('hide');
     }
   }, this));
+
+  var selectInteraction = new ol.interaction.Select({
+    filter: goog.bind(function(feature, layer) {
+      return this.drawnFeatures.getArray().indexOf(feature) != -1;
+    }, this)
```

Comment:
I created a path for that. See https://github.com/openlayers/ol3/pull/4143. Let's see what the other developers say about it.